Source/.editorconfig

```diff
@@ -11,24 +11,65 @@ indent_style = tab
 indent_size = 4
 trim_trailing_whitespace = true
 
-[*.cs]
-indent_size = 4
+# Microsoft .NET properties
+csharp_new_line_before_open_brace = all
+csharp_preferred_modifier_order = private, async, protected, public, internal, volatile, abstract, new, override, sealed, static, virtual, extern, readonly, unsafe, file, required:suggestion
+csharp_preserve_single_line_blocks = true
+dotnet_style_parentheses_in_arithmetic_binary_operators = never_if_unnecessary:none
+dotnet_style_parentheses_in_other_binary_operators = always_for_clarity:none
+dotnet_style_parentheses_in_relational_binary_operators = never_if_unnecessary:none
 dotnet_sort_system_directives_first = true
 dotnet_style_prefer_auto_properties = true
 dotnet_style_prefer_compound_assignment = true
 dotnet_style_coalesce_expression = true
 dotnet_style_prefer_simplified_interpolation = true
-dotnet_diagnostic.CS0168.severity = none
 csharp_style_var_for_built_in_types = true
 csharp_style_var_when_type_is_apparent = true
 
-csharp_preferred_modifier_order = public,private,protected,internal,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,volatile,async:suggestion
+# ReSharper properties
```

Comment:
Not using reshapper.

Reply:
I tihnk it's fine to leave this config here, it allows code contributions from resharper to align with VS